packages/okutrade/src/OkuTrade.ts

```diff
@@ -86,7 +87,10 @@ export const options = async (
   })
 }
 
-export const getSupportedChainIds = async () => {
+export const getSupportedChainIds = async (actionType?: ActionType) => {
```

Comment:
I remember we added this support for another plug-in but we'll want to double check this is still supported on the front-end and behaves nicely/as expected.

Reply:
It was for the paraswap stake action. It is still working correctly showing a different list of networks on swap/stake.